benchmarks/perf-tool/okpt/test/steps/steps.py

```diff
@@ -454,7 +454,8 @@ def _action(self):
         results['took'] = [
             float(query_response['took']) for query_response in query_responses
         ]
-        results['memory_kb'] = get_cache_size_in_kb(self.endpoint, 80)
+        port = 9200 if self.endpoint == 'localhost' else 80
```

Comment:
this fixes exception when running in local, otherwise it tries port 80 but we're running on 9200 by default